node/flatpak_node_generator/providers/npm.py

```diff
@@ -321,6 +324,29 @@ async def generate_package(self, package: Package) -> None:
     def relative_lockfile_dir(self, lockfile: Path) -> Path:
         return lockfile.parent.relative_to(self.lockfile_root)
 
+    @functools.lru_cache(typed=True)
+    def get_lockfile_rc(self, lockfile: Path) -> Dict[str, str]:
+        rc = {}
+        rcfile_paths = []
+        for path in lockfile.parents:
```

Comment:
Afaik npmrc files aren't evaluated up a tree, only yarnrc files?

Reply:
Not sure. Is it documented somewhere? Or any hints in npm source code?

Reply:
I'll just take your word on it. Even if it proves wrong, we can put back the tree lookup code.